docs/navigation-parameters-reference.md

```diff
@@ -96,6 +96,12 @@ The navigation parameters allow you to configure **global** navigation settings
 - **description**: renders all views in new frames. This setting overrides the same-domain frame reuse.
 - **default**: the parameter **defaults.isolateView** is `false` by default, and you can overwrite it using the **isolateView** value on a single node level.
 
+### tooltipText
+- **type**: string
+- **description**: Add tooltip to all of the links and buttons of the left side navigation and top navigation.
+- **default**: the parameter **defaults.tooltipText** is `null` by default, and you can overwrite it using the **tooltipText** value on a single node level.
```

Comment:
``````suggestion
- **description**: allows you to add a tooltip to all of the links and buttons of the left side navigation and top navigation. You can set the parameter with the text that will appear on the tooltip. 
- **example**:
```javascript
tooltipText: 'Useful links'
```
``````
Maybe it's clearer if we add an example on how to use it?